server/src/main/protobuf/FlightsMessage.proto

```diff
@@ -51,11 +51,13 @@ message FlightsDiffMessage {
 }
 
 message FeedStatusesMessage {
-    optional string name = 1;
+    optional string feedSource = 6;
```

Comment:
We could actually drop this field since the messages are stored against an actor that has the feed source defined